### PR TITLE
Adding apple watch

### PR DESCRIPTION
**Pull request by bassrock**
This pull request adds apple watch app icon support and screen shot support

### CODE REVIEW

File: lib/deliver/itunes_connect/itunes_connect_app_icon.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ def upload_app_icon!(app, path)
 
         Helper.log.info "Starting upload of new app icon".green
 
-        evaluate_script("$('.appversionicon > .ios7-style-icon').prev().click()") # delete button
+        evaluate_script("$('.appversionicon:not(.watchIcon) > .ios7-style-icon').prev().click()") # delete button
```
Comment (KrauseFx) [inline lib/deliver/itunes_connect/itunes_connect_app_icon.rb]: Wow, how did you catch this? :+1: Does Apple use the same class?

Comment (KrauseFx) [inline lib/deliver/itunes_connect/itunes_connect_app_icon.rb]: Oh, you to provide a second app icon for the watch, I see.

Comment (bassrock) [inline lib/deliver/itunes_connect/itunes_connect_app_icon.rb]: Yea. With the watch there are two icon files. The current code deletes both, but only uploads the App Icon, not the watch icon. So I created two separate methods for it.

Comment (bassrock) [inline lib/deliver/itunes_connect/itunes_connect_app_icon.rb]: And they added watchIcon to the new Watch icon but it looks like it uploads the same.

**Comment by KrauseFx** [comment]
Looks great, thanks for working on this!

So I see, screenshot upload was straight forward, while the app icon was more tedious. 

Have you tested both the screenshots and the app icon? If so, how can a user define the app icon the `Deliverfile`?

**Comment by bassrock** [comment]
Haven't fully tested it, but yes the screen shot looked straight forward enough when I download my itmsp file and saw it was a new constant. 

As for the app icon it should be as simple as 

```
app_icon "./AppIcon.png"
apple_watch_app_icon "./AppIcon.png"
```
in the ```Deliverfile```

Trying to test it properly with my app now, but experiencing iTunes Connect Connectivity issues.

**Comment by bassrock** [comment]
Just needed a few tweaks but now it works! Surprising I had to open the apple watch section before it could find the upload field.

**Comment by bassrock** [comment]
I believe it is failing due to it not getting the fastlane core change.

**Comment by KrauseFx** [comment]
That's odd. But yes, your pull requests shouldn't be causing any issues. I'll take a closer look.

**Comment by KrauseFx** [comment]
Have you tested the uploads for your project already?

**Comment by bassrock** [comment]
Yep it works great!

**Comment by KrauseFx** [comment]
Great to hear, can you also update the documentation to quickly describe how to use it? https://github.com/KrauseFx/deliver/blob/master/Deliverfile.md

Thanks :+1:

**Comment by bassrock** [comment]
No Problem, documentation updated!

**Comment by KrauseFx** [comment]
Perfect, thank you so much! :+1: This will be in the next release!

**Pull request merged**